app/models/service_template_ansible_playbook.rb

```diff
@@ -117,4 +109,27 @@ def self.validate_config_info(info)
     info
   end
   private_class_method :validate_config_info
+
+  def job_template(action)
+    resource_actions.find_by!(:action => action.to_s.capitalize).configuration_template
+  end
+
+  def update_catalog_item(options, auth_user = nil)
+    config_info = validate_update_config_info(options)
+    name = options[:name]
+    description = options[:description]
+    [:provision, :retirement, :reconfigure].each do |action|
+      next unless config_info[action] && config_info[action].keys.include?(:playbook_id)
```

Comment:
@bzwei - Would there ever be a case where we would create a new Dialog without updating the `JobTemplate`?